db/migrate/20220211010810_create_social_networks.rb

```diff
@@ -1,10 +1,8 @@
 class CreateSocialNetworks < ActiveRecord::Migration[6.1]
   def change
     create_table :social_networks do |t|
-      t.string :twitter
-      t.string :Facebook
-      t.string :LinkendIn
-      t.string :Instagram
+      t.string :url
+      t.string :network
```

Comment:
Good change of table values, some time ago I had made this observation.

Reply:
Done